specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/examples/BillingActivityEventCategory.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "api-version": "2019-10-01-preview",
+    "billingAccountName": "{billingAccountName}"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/7303440/billingActivityEventCategoryListResult",
```

Comment:
these IDs are not appropriate.  To match the API it should be "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingActivityEventCategory/7303440"